### PR TITLE
712 am1 part 7 new address telephone capture not ind

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

Instead of using treatment code to derive the what the qid type is, we are now using case type and region instead.

# What has changed

- Calculate qid type by Case Type and Region

# How to test?

- Run mvn clean install
- Run with other branches and run the ATs

# Links

[Trello](https://trello.com/c/5NVWoFG9)
https://github.com/ONSdigital/census-rm-acceptance-tests/pull/228

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[AM1 part 7 - New Address - Telephone Capture - not ind response (8)](https://trello.com/c/5NVWoFG9/712-am1-part-7-new-address-telephone-capture-not-ind-response-8)

File: src/main/java/uk/gov/ons/census/casesvc/utility/QuestionnaireTypeHelper.java [hunk 1]
```diff
@@ -91,8 +102,19 @@ public static boolean isIndividualQuestionnaireType(String questionnaireId) {
...
 
-  public static boolean iscontinuationQuestionnaireTypes(String questionnaireId) {
-    String questionnaireType = questionnaireId.substring(0, 2);
-    return continutationQuestionnaireTypes.contains(questionnaireType);
+  private static boolean isCE1RequestForEstabCeCase(String caseType, String addressLevel) {
+    return isCeCaseType(caseType) && addressLevel.equals(ADDRESS_LEVEL_ESTAB);
+  }
+
+  private static boolean isSpgCaseType(String caseType) {
+    return caseType.startsWith(CASE_TYPE_SPG);
```
Comment (NickGrantONS) [inline src/main/java/uk/gov/ons/census/casesvc/utility/QuestionnaireTypeHelper.java:110]:
```suggestion
    return caseType.equals(CASE_TYPE_SPG);
```

File: src/main/java/uk/gov/ons/census/casesvc/utility/QuestionnaireTypeHelper.java [hunk 2]
```diff
@@ -91,8 +102,19 @@ public static boolean isIndividualQuestionnaireType(String questionnaireId) {
...
+  }
+
+  private static boolean isHouseholdCaseType(String caseType) {
+    return caseType.startsWith(CASE_TYPE_HOUSEHOLD);
```
Comment (NickGrantONS) [inline src/main/java/uk/gov/ons/census/casesvc/utility/QuestionnaireTypeHelper.java:114]:
```suggestion
    return caseType.equals(CASE_TYPE_HOUSEHOLD);
```

File: src/main/java/uk/gov/ons/census/casesvc/utility/QuestionnaireTypeHelper.java [hunk 3]
```diff
@@ -91,8 +102,19 @@ public static boolean isIndividualQuestionnaireType(String questionnaireId) {
...
+  }
+
+  private static boolean isCeCaseType(String caseType) {
+    return caseType.startsWith(CASE_TYPE_CE);
```
Comment (NickGrantONS) [inline src/main/java/uk/gov/ons/census/casesvc/utility/QuestionnaireTypeHelper.java:118]:
```suggestion
    return caseType.equals(CASE_TYPE_CE);
```

File: src/main/java/uk/gov/ons/census/casesvc/utility/QuestionnaireTypeHelper.java (hunk 1 above)
Comment (ryangrundy7) [reply src/main/java/uk/gov/ons/census/casesvc/utility/QuestionnaireTypeHelper.java]: Fixed 👍

File: src/main/java/uk/gov/ons/census/casesvc/utility/QuestionnaireTypeHelper.java (hunk 3 above)
Comment (ryangrundy7) [reply src/main/java/uk/gov/ons/census/casesvc/utility/QuestionnaireTypeHelper.java]: Fixed 👍

File: src/main/java/uk/gov/ons/census/casesvc/utility/QuestionnaireTypeHelper.java (hunk 2 above)
Comment (ryangrundy7) [reply src/main/java/uk/gov/ons/census/casesvc/utility/QuestionnaireTypeHelper.java]: Fixed 👍

**Pull request merged**